ui/app/mainui/activitycenter/panels/MembershipCta.qml

```diff
@@ -14,6 +15,9 @@ Item {
     property bool pending: false
     property bool accepted: false
     property bool declined: false
+    property bool acceptedPending: false
```

Comment:
I know this is not new and you just followed the existing code but, I'm wondering if it makes sense to update all these state properties to be `readonly` and have only a new property called `membershipStatus` coming from outside. Something like this:
```
property int membershipStatus

readonly property int pending: membershipStatus === ActivityCenterStore.ActivityCenterMembershipStatus.Pending
readonly property int accepted: notification.membershipStatus === ActivityCenterStore.ActivityCenterMembershipStatus.Accepted
...

```

Reply:
good point!